gp-ant-task/src/main/java/com/ibm/g11n/pipeline/ant/GPBaseTask.java

```diff
@@ -234,14 +233,14 @@ public BundleSet createBundleSet() {
      * @param path  Relative path to package root
      * @return A bundle ID corresponding to the resource type and path.
      */
-    private String pathToBundleId(ResourceType type, String path) {
+    private String pathToBundleId(String type, String path) {
         File f = new File(path);
         File parent = f.getParentFile();
         String pkgName = parent == null ? "" :
             parent.getPath().replace(File.separatorChar, '.');
 
         String fileName = f.getName().replaceAll(" ", "_");
-        if (type == ResourceType.JAVA) {
+        if (type.equals("JAVA")) {
```

Comment:
change constant to `DefaultResourceFilterProvider.Filter.JAVA.name()`?

Reply:
is this also an 'id' where type is single or multi

Reply:
Same as above.

I was thinking about both single bundle and multi bundle filter implementation can be provided through a single resource filter ID. But I decided not to do so.

So, for a given filter ID "foo", it is either a single bundle filter or a multiple bundle filter, but cannot do both.

Reply:
also never mind, this is not the filter id but bundle type within ant.  

Ignore this comment ^